DFC.TestAutomation.UI/BrowserType.cs

```diff
@@ -0,0 +1,32 @@
+﻿// <copyright file="BrowserType.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
```

Comment:
Update with relevant information or remove.

Reply:
Done. I have put National Careers Service in place of PlaceholderCompany